src/rust/ide/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -168,6 +172,7 @@ pub struct View {
     display_object : display::object::Instance,
     background     : component::ShapeView<background::Shape>,
     overlay        : component::ShapeView<overlay::Shape>,
+    dom            : DomSymbol
```

Comment:
the `dom` alone is not telling much, rename it to "background_dom" and add TODO refering to those TODOs above (with issue link as well).